### PR TITLE
[FEATURE] notiQueue size append in the report

### PR DESCRIPTION
**Pull request by iariasleon**
@fgalan @kzangeli @crbrox 

```
   Show requests x seconds (TPS)... [CTRL+C] to stop!
----------------------------------------------------------------------------
                  reqs       requests      tps from first    notifQueue
     seconds    each sec      total        to last request      size
-----------------------------------------------------------------------------
--- [ 1 ] ----- [ 34 ] ----- [ 34 ] ----- [ 34.0/s ] ------- [ 793 ]
--- [ 2 ] ----- [ 24 ] ----- [ 58 ] ----- [ 29.0/s ] ------- [ 824 ]


```

### CODE REVIEW

File: test/loadTest/cache_refresh/reqs_x_secs.sh [hunk 1]
```diff
@@ -56,16 +88,28 @@ while true
...
+  # requests received in the listener
+  resp=`curl -s $LISTENER/receive 2>&1`
   total=`echo $resp | sed 's/^{"requests": "\(.*\)","tps":\(.*\)"}/\1/'`
   tps=`echo $resp | sed 's/^{"requests": "\(.*\)","tps": "\(.*\)"}/\2/'`
+
+  # notifQueue size
+  if [ "$CB" != "None" ]
+    then
+      stat=`curl -s $CB/statistics  2>&1`
+      stat1=`echo $stat | grep -Eo 'size":[[:digit:]]+'`
```
Comment (fgalan) [inline test/loadTest/cache_refresh/reqs_x_secs.sh]: Not a very good idea to parse a JSON as it were a string text... However, maybe we doesn't have any option from bash.

Something like `jq` command would be greate (https://stedolan.github.io/jq). However probably it is not very much extended...

Comment (iariasleon) [inline test/loadTest/cache_refresh/reqs_x_secs.sh]: Fixed in  	1de65fc

File: test/loadTest/cache_refresh/README.md [hunk 2]
```diff
@@ -74,6 +74,16 @@ These tests are used to verify the cache refresh in a balanced system.
...
   ```
   
+  Dependency:
+  ```
+  jq is a lightweight and flexible command-line JSON processor. It is used for the json with statistics returned from ContextBroker.
+       steps to install jq:
+      ---------------------
+           wget http://stedolan.github.io/jq/download/linux64/jq
+           chmod +x ./jq
+           cp jq /usr/bin
```
Comment (fgalan) [inline test/loadTest/cache_refresh/README.md]: Locally installed stuff should be at /usr/local/bin. In addition, you need root privileges to copy there, so probably is better to say:

```
sudo cp jq /usr/local/bin
```

**Comment by fgalan** [comment]
LGTM. Passing the ball to @kzangeli (whose bash-lore is greater than mine by large :)  for final LGTM + merge + delete branch

Note it is safe to merge during the "1.6 releasea candidate under soak" period, as this PR doesn't involves changes in the CB code.

File: test/loadTest/cache_refresh/reqs_x_secs.sh [hunk 3]
```diff
@@ -21,33 +21,68 @@
...
+     exit 0
+}
+
+# BEGIN
+while [ "$#" != 0 ]
+do
+  if   [ "$1" == "-u" ];          then usage;
+  elif [ "$1" == "--listener" ];  then LISTENER="$2"; shift;
+  elif [ "$1" == "--cb" ];        then CB="$2"; shift;
+  elif [ "$1" == "-reset" ];      then RESET="true"; shift;
```
Comment (kzangeli) [inline test/loadTest/cache_refresh/reqs_x_secs.sh]: This `shift` will eat a parameter.
Not what you want.

[ it works *only* if -reset is the *last* parameter ]

Comment (iariasleon) [inline test/loadTest/cache_refresh/reqs_x_secs.sh]: fixed in d3fbbea

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**